### PR TITLE
docs(installation): add instructions for NPM usage

### PR DESCRIPTION
**Pull request by xinbenlv**
Add instructions for NPM

### CODE REVIEW

File: docs/cli/installation.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ yarn add --dev all-contributors-cli # or npm
 Init the project using `init` and answer a few questions
 
 ```console
-yarn all-contributors init # npm run
+yarn all-contributors init # or npx all-contributors init
```
Comment (Berkmann18) [inline docs/cli/installation.md:27]: Is there any tangible advantage in using `npx` over `npm run` in scenarios where one would use the AC CLI?

Comment (xinbenlv) [reply docs/cli/installation.md:27]: `npm run` is not equivalent to `yarn all-contributors init`. 

`npx all-contributors init` is equivalent to it instead.

Comment (Berkmann18) [reply docs/cli/installation.md:27]: That makes sense.

**Comment by Berkmann18** [comment]
@all-contributors please add @xinbenlv for docs

**Pull request merged**